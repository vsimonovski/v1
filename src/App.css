html,
body {
  height: 100%;
  font-size: 10px;
}

#root,
.App {
  height: 100%;
}
