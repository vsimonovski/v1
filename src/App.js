import React from 'react';
import './App.css';
import 'normalize.css';
import 'wenk';
import Title from './components/title/Title';
import About from './components/about/About';
import Contact from './components/contact/Contact';
import { ThemeProvider } from 'styled-components';
import { Theme } from './Theme';

function App() {
  return (
    <ThemeProvider theme={Theme}>
      <div className="App">
        <div className="content">
          <div className="content__inner">
            <Title />
            <About />
            <Contact />
          </div>
        </div>
      </div>
    </ThemeProvider>
  );
}

export default App;
