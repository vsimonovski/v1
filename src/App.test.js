import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

it('should match snapshot', () => {
  const tree = render(<App />);
  expect(tree).toMatchSnapshot();
});
