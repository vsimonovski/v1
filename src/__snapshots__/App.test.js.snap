// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="sc-fzpans hJAUgc"
        >
          <div
            class="sc-fzplWN jVRtJf"
          >
            <section>
              <h1
                class="sc-AxhCb leQIUN"
              >
                <span
                  class="sc-AxhUy flUQeB"
                />
                Hi, I'm Vladimir
                <span
                  class="sc-AxjAm iSVuRx"
                />
              </h1>
              <br />
              <h1
                class="sc-AxhCb leQIUN"
              >
                I
                <span
                  aria-label="heart"
                  class="sc-AxirZ sc-AxiKw cUGkJy"
                  role="img"
                >
                  ‚ù§Ô∏è
                </span>
                Web
                <span
                  class="sc-AxjAm iSVuRx"
                />
              </h1>
            </section>
            <section
              class="sc-AxgMl gzSaJe"
            >
              <span
                class="sc-AxheI iiQIKH"
                data-wenk="An organism that converts  ‚òï  into code."
              >
                JavaScript enthusiast
              </span>
               
              with strong comp. sci. background.
              <br />
               Passionate about code quality, best practices, security, performance and everything web related.
            </section>
            <section
              class="sc-Axmtr NImwC"
            >
              <div
                class="sc-AxmLO gHYGBN"
              >
                <span
                  class="sc-fzozJi kmvZGQ"
                >
                  üë®‚Äçüíª
                </span>
                <a
                  class="sc-fzoLsD exAFxw"
                  data-wenk="üëà  I work here"
                  data-wenk-pos="right"
                  href="https://www.linkedin.com/in/vsimonovski/"
                >
                  @Hooloovoo
                </a>
              </div>
              <div
                class="sc-AxmLO gHYGBN"
              >
                <span
                  class="sc-fzozJi kmvZGQ"
                >
                  üíª
                </span>
                <a
                  class="sc-fzoLsD exAFxw"
                  data-wenk="üëà  Check out my GitHub"
                  data-wenk-pos="right"
                  href="https://github.com/vsimonovski"
                >
                  @vsimonovski
                </a>
              </div>
              <div
                class="sc-AxmLO gHYGBN"
              >
                <span
                  class="sc-fzozJi kmvZGQ"
                >
                  üóíÔ∏è
                </span>
                <a
                  class="sc-fzoLsD exAFxw"
                  data-wenk="üëà  Sometimes I like to write about code"
                  data-wenk-pos="right"
                  href="http://blog.vsimonovski.me/"
                >
                  @blog.vsimonovski
                </a>
              </div>
              <div
                class="sc-AxmLO gHYGBN"
              >
                <span
                  class="sc-fzozJi kmvZGQ"
                >
                  üíå
                </span>
                <a
                  class="sc-fzoLsD exAFxw"
                  href="mailto:vsimonovski@outlook.com"
                >
                  Contact me?
                </a>
              </div>
            </section>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="sc-fzpans hJAUgc"
      >
        <div
          class="sc-fzplWN jVRtJf"
        >
          <section>
            <h1
              class="sc-AxhCb leQIUN"
            >
              <span
                class="sc-AxhUy flUQeB"
              />
              Hi, I'm Vladimir
              <span
                class="sc-AxjAm iSVuRx"
              />
            </h1>
            <br />
            <h1
              class="sc-AxhCb leQIUN"
            >
              I
              <span
                aria-label="heart"
                class="sc-AxirZ sc-AxiKw cUGkJy"
                role="img"
              >
                ‚ù§Ô∏è
              </span>
              Web
              <span
                class="sc-AxjAm iSVuRx"
              />
            </h1>
          </section>
          <section
            class="sc-AxgMl gzSaJe"
          >
            <span
              class="sc-AxheI iiQIKH"
              data-wenk="An organism that converts  ‚òï  into code."
            >
              JavaScript enthusiast
            </span>
             
            with strong comp. sci. background.
            <br />
             Passionate about code quality, best practices, security, performance and everything web related.
          </section>
          <section
            class="sc-Axmtr NImwC"
          >
            <div
              class="sc-AxmLO gHYGBN"
            >
              <span
                class="sc-fzozJi kmvZGQ"
              >
                üë®‚Äçüíª
              </span>
              <a
                class="sc-fzoLsD exAFxw"
                data-wenk="üëà  I work here"
                data-wenk-pos="right"
                href="https://www.linkedin.com/in/vsimonovski/"
              >
                @Hooloovoo
              </a>
            </div>
            <div
              class="sc-AxmLO gHYGBN"
            >
              <span
                class="sc-fzozJi kmvZGQ"
              >
                üíª
              </span>
              <a
                class="sc-fzoLsD exAFxw"
                data-wenk="üëà  Check out my GitHub"
                data-wenk-pos="right"
                href="https://github.com/vsimonovski"
              >
                @vsimonovski
              </a>
            </div>
            <div
              class="sc-AxmLO gHYGBN"
            >
              <span
                class="sc-fzozJi kmvZGQ"
              >
                üóíÔ∏è
              </span>
              <a
                class="sc-fzoLsD exAFxw"
                data-wenk="üëà  Sometimes I like to write about code"
                data-wenk-pos="right"
                href="http://blog.vsimonovski.me/"
              >
                @blog.vsimonovski
              </a>
            </div>
            <div
              class="sc-AxmLO gHYGBN"
            >
              <span
                class="sc-fzozJi kmvZGQ"
              >
                üíå
              </span>
              <a
                class="sc-fzoLsD exAFxw"
                href="mailto:vsimonovski@outlook.com"
              >
                Contact me?
              </a>
            </div>
          </section>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
